src/plots/gl2d/scene2d.js

```diff
@@ -286,10 +292,11 @@ proto.updateFx = function(options) {
 
     fullLayout.dragmode = options.dragmode;
     fullLayout.hovermode = options.hovermode;
-};
 
-var relayoutCallback = function(scene) {
+    this.graphDiv._fullLayout = fullLayout;
```

Comment:
alright, well I'm not entirely sure what would happen, but the test would be something like:
- do a relayout that causes a new `gd._fullLayout` to be constructed by `supplyDefaults` but DOESN'T result in a new `scene2d.plot` (like what? change legend background color? change something cosmetic on an axis of a different subplot?)
- trigger this code by changing dragmode
- do something else that causes a full plot redraw (maybe just `Plotly.redraw(gd)`)
- does the initial relayout get reverted?